devtools/cli/src/main/java/io/quarkus/cli/common/CategoryListFormatOptions.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.cli.common;
+
+import picocli.CommandLine;
+
+public class CategoryListFormatOptions {
+    @CommandLine.Option(names = { "--name" }, order = 1, description = "Display category name only. (default)")
```

Comment:
I was thinking if `--name` format switch should rather be `--id`, both for extensions and categories (and the same for maven and gradle). Cause it doesn't really show verbose names but artifactIds / category ids. It feels bit confusing to me, plus I don't what to write in the description - "`--name` displays category id". :)

Reply:
@aloubyansky  ?  I don't have an issue with switching to `--id`